helm/values.yaml

```diff
@@ -42,7 +42,7 @@ secret:
 
 # appSettings:
 #   env:
-#    FIFTYONE_DATABASE_ADMIN is set to `false` by default for v1.5.3 installs
+#    FIFTYONE_DATABASE_ADMIN is set to `false` by default for v1.5.4 installs
```

Comment:
I look forward to when we remove some of version strings peppered through the repo.  Some don't provide meaningful contexts.

Reply:
seems like the best way to do that is for you to make a PR that removes the ones you don't think are important...